sarahmarie1976/Microblog
# :page_facing_up: created profile page

Summary:
This PR enhances user profiles by allowing profile picture uploads and expanding the "About Me" section. It also includes styling improvements and necessary database migrations.

Changes:
Modified Files:

app/__init__.py: Initialized file upload configuration.
app/forms.py: Added profile_picture field to EditProfileForm.
app/models.py: Updated User model to include profile_picture and extended about_me field length. Changed body field in Post to text.
app/routes.py: Added logic for profile picture uploads and profile updates.
app/templates/base.html: Linked new CSS file.
config.py: Added file upload configuration.
New Files:

app/templates/404.html
app/templates/_post.html
app/templates/edit_profile.html
app/templates/user.html
static/styles.css
uploads/IMG_5646.jpeg
Migrations:

Added migrations for new fields and schema changes.
How to Test:
Profile Picture:
Edit profile, upload a picture, save, and verify on profile page.
About Me Section:
Edit profile, update "About Me", save, and verify on profile page.
Styling:
Check the profile page and posts for correct layout and styling.
